vmdb/spec/requests/api/tag_collections_spec.rb

```diff
@@ -29,441 +43,361 @@ def app
   end
 
   context "Provider Tag subcollection" do
-    before(:each) do
-      @provider          = @ems
-      @provider_url      = "#{@cfme[:providers_url]}/#{@ems.id}"
-      @provider_tags_url = "#{@provider_url}/tags"
+    let(:provider)          { ems }
+    let(:provider_url)      { providers_url(provider.id) }
```

Comment:
should this be `provider_url(provider.id)`? or just plain `provider_url(provider)`